<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Registration</class>
 <widget class="QWidget" name="Registration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="loginLineEdit">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>45</y>
     <width>400</width>
     <height>35</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordLineEdit">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>125</y>
     <width>400</width>
     <height>35</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameLineEdit">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>205</y>
     <width>400</width>
     <height>35</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="surnameLineEdit">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>285</y>
     <width>400</width>
     <height>35</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="patronymicLineEdit">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>365</y>
     <width>400</width>
     <height>35</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="homeLineEdit">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>445</y>
     <width>400</width>
     <height>35</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="enterButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>540</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>ВОЙТИ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="regButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>540</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>ЗАРЕГИСТРИРОВАТЬСЯ</string>
   </property>
  </widget>
  <widget class="QLabel" name="loginLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>121</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Логин</string>
   </property>
  </widget>
  <widget class="QLabel" name="passwordLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>120</y>
     <width>181</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Пароль</string>
   </property>
  </widget>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>200</y>
     <width>141</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Имя</string>
   </property>
  </widget>
  <widget class="QLabel" name="surnameLabel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>280</y>
     <width>151</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Фамилия</string>
   </property>
  </widget>
  <widget class="QLabel" name="patronymicLabel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>360</y>
     <width>151</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Отчество</string>
   </property>
  </widget>
  <widget class="QLabel" name="homeLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>440</y>
     <width>231</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Домашний адрес</string>
   </property>
  </widget>
  <widget class="QLabel" name="errorLable">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>500</y>
     <width>400</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>regButton</sender>
   <signal>clicked()</signal>
   <receiver>Registration</receiver>
   <slot>on_accept_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>560</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_accept_clicked()</slot>
 </slots>
</ui>
